sccm/desktop-analytics/tutorial-windows10.md

```diff
@@ -219,6 +219,13 @@ Install the Configuration Manager version 1902 update rollup (4500571) to suppor
 
 Configuration Manager creates a settings policy to configure devices in the Target Collection. This policy includes the diagnostic data settings to enable devices to send data to Microsoft. By default, clients update policy every hour. After receiving the new settings, it can be several hours more before the data is available in Desktop Analytics.
 
+>[!Note]  
+>To confirm machines settings by Desktop Analytics refer to the following policy key: 
```

Comment:
Can we change this to instead just point to the following section?

```suggestion
> For more information on these settings, see [Windows settings](/sccm/desktop-analytics/enroll-devices#windows-settings).
```

(and then nix the additional lines of the note)

Reply:
Yes, agreed....